cli/cmdlineparser.cpp

```diff
@@ -399,7 +398,7 @@ bool CmdLineParser::parseFromArgs(int argc, const char* const argv[])
                     path = Path::fromNativeSeparators(path);
                     path = Path::simplifyPath(path);
 
-                    if (FileLister::isDirectory(path)) {
```

Comment:
hmm.. sorry. I think I gave the wrong advice about putting `folderExists` in `Path`. It would be preferable to keep the old code in `FileLister` and reuse `FileLister::isDirectory`. Sorry I should have realized this from the start.

Reply:
Reverted my changes FileLister:fileExists Windows implimentation was broken i fixed it.